package.json

```
@@ -34,7 +35,7 @@
   "files": [
     "LICENSE",
     "README.md",
-    "/lib"
+    "lib/"
```

```suggestion
    "/lib"
```

I am confused is it `/lib` or `lib/`?
https://github.com/asyncapi/bundler/pull/76/files changed it to `/lib`, and also cli uses https://github.com/asyncapi/cli/blob/bf66a0c58362c2f8012e5285b6f4bf8ace26311d/package.json#L84 so lets just leave it as `/lib`

---

You can run `npm run build && npm pack` and check if `/lib` folder is inside packed package :)

---

> /lib and lib/ works, so the only concern is consistency really

I remember that some version in the parser v2 did not work 😅 so we should check that by scripts `npm run build && npm pack`.